components/Button/index.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+
+export const SIZE_CLASSES = {
+  xs: 'w-[90px] h-10',
+  small: 'w-30 h-14',
+  medium: 'w-80 h-14',
+  large: 'w-[327px] h-14',
+  huge: 'w-176 h-14',
+};
+
+const Button = ({ size, dark, children, ...attributes }) => {
```

Comment:
should we set a default value to the `dark` prop? :) 

This way, we can use it like a `required` or `disabled` attribute (no need to specify the value, if the attribute is present, it is true. Else it is false) ;-) 

```suggestion
const Button = ({ size, dark = false, children, ...attributes }) => {
```
(not tested!!)

Reply:
And maybe set it as an optional prop 💭  `dark: PropTypes.bool,`

Reply:
done in https://github.com/tyrro/react-survey/commit/dbf9d1e261944188cc87b0d6f495a313bea9b890